cmd/libs/go2idl/types/types.go

```diff
@@ -112,6 +123,28 @@ func (p *Package) Get(typeName string) *Type {
 	return t
 }
 
+// Get (or add) the given function
+func (p *Package) Function(funcName string) *Type {
+	if t, ok := p.Functions[funcName]; ok {
+		return t
+	}
+	t := &Type{Name: Name{Package: p.Path, Name: funcName}}
+	t.Kind = DeclarationOf
+	p.Functions[funcName] = t
+	return t
+}
+
+// Get (or add) the given function
+func (p *Package) Variable(varName string) *Type {
+	if t, ok := p.Variables[varName]; ok {
+		return t
+	}
+	t := &Type{Name: Name{Package: p.Path, Name: varName}}
+	t.Kind = DeclarationOf
+	p.Variables[varName] = t
+	return t
+}
+
```

Comment:
...actually, maybe save it for another PR so we don't get bogged down adding an "untyped" builtin :)